chanzuckerberg/edu-design-system
# fix: link deprecated border token to new token value

### Summary:

- if we use the style-dictionary syntax, it would copy the raw value into the token
- instead use the CSS syntax to reference the value from the deprecated token
- we do this so that any existing themes using that token will continue to work
- if pepole theme the new token directly, it will also work (the components use the new token)

Caveat: if a project uses the old token in custom components, they would need to update both. We update the code comment to mention the desired token to use.

### Test Plan:


  - use storybook to test token overrides of the old and new token to verify behavior
  - use storybook snapshots to verify there are no undesirable snapshot changes
